diff -up cdrkit-1.1.9/genisoimage/genisoimage.c.graftpoints cdrkit-1.1.9/genisoimage/genisoimage.c
--- cdrkit-1.1.9/genisoimage/genisoimage.c.graftpoints	2012-03-05 16:21:49.339287083 +0100
+++ cdrkit-1.1.9/genisoimage/genisoimage.c	2012-03-05 16:22:13.297987559 +0100
@@ -3141,7 +3141,7 @@ if (check_session == 0)
 				while (*xpnt == PATH_SEPARATOR) {
 					xpnt++;
 				}
-				strcpy(graft_point, xpnt);
+				memmove(graft_point, xpnt, strlen(xpnt)+1);
 			} while (xpnt > graft_point);
 
 			if (node) {
