diff -up cdrkit-1.1.11/libusal/scsi-linux-sg.c.sfdc01983864 cdrkit-1.1.11/libusal/scsi-linux-sg.c
--- cdrkit-1.1.11/libusal/scsi-linux-sg.c.sfdc01983864	2017-12-12 08:55:08.000000000 +0100
+++ cdrkit-1.1.11/libusal/scsi-linux-sg.c	2017-12-12 09:00:57.000000000 +0100
@@ -251,7 +251,7 @@ static BOOL get_max_secs(char *dirpath,
 BOOL check_linux_26() {
 	int gen, tmp;
 	struct utsname buf;
-	return ( 0==uname( &buf ) && sscanf(buf.release, "%d.%d", &gen, &tmp)>1 && tmp>=6);
+	return ( 0==uname( &buf ) && sscanf(buf.release, "%d.%d", &gen, &tmp)>1 && tmp>=6 && gen == 2);
 }
 
 int sg_open_excl(char *device, int mode, BOOL beQuiet)
